packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -886,3 +886,379 @@ RAW_TOOLS_DIR: ${Env:RAW_TOOLS_DIR}
     VM-Write-Log "INFO" "$($survey -f $osInfo, $memInfo, $diskInfo, $avInfoFormatted, $psInfo, $psInfoClr, $chocoInfo, $boxstarerInfo, $installedPackages, $envVars)"
 }
 
+function VM-Remove-Appx-Package {
+# Function for removing Apps
+    param(
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$appName
+    )
+
+    try {
+        # Check if the app is installed
+        $installedPackage = Get-AppxPackage -Name $appName -ErrorAction SilentlyContinue
+        
+        if ($installedPackage) {
+            $packageFullName = $installedPackage.PackageFullName
+            $result = Remove-AppxPackage -Package $packageFullName -ErrorAction Stop
```

Comment:
Do we want "-ErrorAction stop"? I believe this will cause it to throw an exception and it'll print out the try/catch generic message versus letting us know it failed to remove the installed app.

Reply:
I think you might be right, the logs have not been informative.